canvas_modules/common-canvas/package.json

```diff
@@ -23,8 +23,8 @@
     "@carbon/icons-react": "10.13.0",
     "@carbon/themes": "10.14.0",
     "@elyra/pipeline-schemas": "^3.0.20",
-    "carbon-components": "10.14.0",
-    "carbon-components-react": "7.14.0",
+    "carbon-components": "10.19.0",
+    "carbon-components-react": "7.19.0",
     "carbon-icons": "7.0.7",
     "codemirror": "5.55.0",
     "d3": "5.16.0",
```

Comment:
We'll need to update common-canvas to a new minor version as well.

Reply:
updated to 9.1.0